core/src/components/toolbar/toolbar.ionic.scss

```diff
@@ -31,8 +23,7 @@ $toolbar-order-ionic: (
 // --------------------------------------------------
 
 .toolbar-content {
-  flex: 2;
-  order: map-get($toolbar-order-ionic, content);
+  flex: 1 1 auto;
 
   min-width: 0;
 }
```

Comment:
```suggestion
}

::slotted(ion-buttons) {
  gap: globals.$ion-space-200;
}
```

This will give us the gap between the buttons:

![Screenshot 2025-04-29 at 3 27 58 PM](https://github.com/user-attachments/assets/086cbb8b-f2c9-4e6a-be20-a8176a5c3d08)

Alternatively we can add this to `buttons.ionic.scss`.

Reply:
Done through the toolbar.ionic.scss